APIs/SentinelHub/Data.qmd

```diff
@@ -0,0 +1,13 @@
+---
+title: Data
+---
+
+Some of the data collections available in the Copernicus Data Space
+Ecosystem are indexed (or we could say "imported") in Sentinel Hub,
+which means you can use Sentinel Hub APIs to work with these data
+collections. Each of the data collections available in Sentinel Hub
+has its own subchapter below, which describes how the data is
+pre-processed and how you can access it with Sentinel Hub. For a
+general description of these data collections and related satellite
+missions, please refer to the general [Copernicus Data Space Ecosystem
+Data chapter](/Data.qmd).
```

Comment:
I think the page in look a bit empty https://eu-cdse.github.io/documentation/pr-preview/pr-306/APIs/SentinelHub/Data.html

How about implementing grid view that will display and link all the pages inside the SentinelHub/Data/ ?
We plan to implement similar visualisation to the rest of the section introduction page. 


```suggestion
---
title: Data
listing:
  type: grid
  id: Sentinelhub_data
  contents:
    - Data
  sort: "date desc"
  filter-ui: true
  page-size: 6

---



Some of the data collections available in the Copernicus Data Space
Ecosystem are indexed (or we could say "imported") in Sentinel Hub,
which means you can use Sentinel Hub APIs to work with these data
collections. Each of the data collections available in Sentinel Hub
has its own subchapter below, which describes how the data is
pre-processed and how you can access it with Sentinel Hub. For a
general description of these data collections and related satellite
missions, please refer to the general [Copernicus Data Space Ecosystem
Data chapter](/Data.qmd).


::: {#Sentinelhub_data}
:::
```

However, on each of those pages, you will also need to add image:"imageaddress" under title for thumbnail

Reply:
Other than this, the PR looks good to me.

Reply:
Thats a great feature I didn't now :) I don't think images will provide nicer content, so I modified a bit: https://github.com/eu-cdse/documentation/pull/306/commits/0d346b91d32366d1b207586cbdc2beb1327ef9e2